float64 std_dev_yaw
float64 std_dev_lin_v
float64 avg_lin_v
float64 dist_tot
float64 dist_tot1
float64 time
float64 idle_time
float64 min_range


	
